### ISSUE TITLE
unblocked.id

### ISSUE THREAD

**Issue by chirag127** [AdguardTeam/AdguardFilters#89720]
**Issue URL (Ads)** : https://unblocked.id/

<details><summary>Screenshots:</summary>

![image](https://user-images.githubusercontent.com/76880977/127739294-d8718246-7712-4516-910d-c9eeb8618679.png)

</details><br/>

<details><summary>System configuration:</summary>

![image](https://user-images.githubusercontent.com/76880977/126797496-0434116b-fc94-42c9-8139-fba02a985cf2.png)
</details><br/>

**Comment by AdamWr** [comment]
Looks like duplicate of https://github.com/AdguardTeam/AdguardFilters/issues/89597
